### PR TITLE
libmynteye-release: 0.0.1-0 in 'kinetic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by harjeb**
Increasing version of package(s) in repository `libmynteye-release` to `0.0.1-0`:

- upstream repository: https://github.com/harjeb/libmynteye.git
- release repository: https://github.com/harjeb/libmynteye-release.git
- distro file: `kinetic/distribution.yaml`
- bloom version: `0.6.6`
- previous version for package: `null`

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -5044,6 +5044,14 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2016.4.24-0
     status: maintained
+  libmynteye-release:
```
Comment (clalancette) [inline kinetic/distribution.yaml]: It's a bit strange for the name of the repository to be `libmynteye-release`; this should just be `libmynteye`.

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -5044,6 +5044,14 @@ repositories:
...
+    release:
+      packages:
+      - libmynteye
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/harjeb/libmynteye-release.git
+      version: 0.0.1-0
```
Comment (clalancette) [inline kinetic/distribution.yaml]: I see that you don't have a `doc` or `source` section for this package.  Neither is required, but they can give more information to your users (`doc` will generate documentation for hosting on the wiki, and `source` will allow the build-farm to build the package from source, and do pull request testing).  There's some more information about Pull request testing here http://wiki.ros.org/buildfarm/Pull%20request%20testing

File: kinetic/distribution.yaml [hunk 3]
```diff
@@ -5044,14 +5044,24 @@ repositories:
...
+      url: https://github.com/harjeb/libmynteye.git
+      version: master
     release:
       packages:
       - libmynteye
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/harjeb/libmynteye-release.git
-      version: 0.0.1-0
+      version: 0.0.1-1
```
Comment (wjwwood) [inline kinetic/distribution.yaml]: There's no `0.0.1-1` in the release repository (https://github.com/harjeb/libmynteye-release), you can't just change this you need to rerun bloom.

I'd suggest closing this pull request, rerunning bloom (with the correct repository name, use `-e` to edit the settings before running if necessary) and letting bloom open a new pull request for you.

File: kinetic/distribution.yaml (hunk 2 above)
Comment (harjeb) [reply kinetic/distribution.yaml]: OK, I will close this pull request,  and rerun bloom

**Pull request closed without merging**